package.json

```diff
@@ -2,11 +2,16 @@
   "name": "proposal-flatmap",
   "version": "1.0.0",
   "description": "",
-  "main": "ecmarkup.js",
+  "main": "polyfill.js",
   "dependencies": {
-    "ecmarkup": "^3.11.5"
+    "es-abstract": "^1.7.0"
+  },
+  "devDependencies": {
+    "ecmarkup": "^3.11.5",
+    "eslint": "^4.2.0",
```

Comment:
eslint v4 is not supported by `eslint-config-airbnb-base`; `npm ls` will fail.

Run the install command in the readme for the airbnb config and it'll install the proper peer dep versions.